test/system/libs/helpers.bash

```diff
@@ -551,6 +551,11 @@ function get_system_id() (
 function get_system_version() (
   local os_release
 
+  if [ "$(get_system_id)" = "arch" ]; then
+    echo "latest"
```

Comment:
I needed to solve the same problem that this hunk addresses for https://github.com/containers/toolbox/issues/1438 which reminded me again of this pull request.

Instead of calling `get_system_id`, we can directly use `VERSION_ID` because we also source `os-release(5)` here.